cms/static/js/factories/register.js

```diff
@@ -29,5 +29,30 @@ define(['jquery', 'jquery.cookie'], function($) {
                 }
             });
         });
+
+        $('input#password').blur(function() {
+            // Uninitialize the errors on blur
+            var $formErrors = $('#password_error');
+            $formErrors.empty();
+            $formErrors.addClass('hidden');
+
+            var data = {
+                password: $('#password').val()
+            };
+            $.ajax({
+                url: '/api/user/v1/validation/registration',
+                type: 'POST',
+                dataType: 'json',
+                data: data,
+                success: function(json) {
+                    _.each(json.validation_decisions, function(value, key) {
```

Comment:
Boo underscore. But I get that this isn't es6 code yet.

Could avoid the loop by doing something like 'if (json.validation_decisions && json.validation_decisions.password)' maybe?

Reply:
Reason I did the loop is because validation_decisions comes back as an `[]`. I figured there are scenarios that exist that could have multiple `password` keyed errors and we would want to include them as well.

Underscore comment... fair point, but just keeping consistent with the rest of the file.